### ISSUE TITLE
Filenames with commas

### ISSUE THREAD

**Issue by peterbe** [uams/geturl#16]
```
peterbe@host-7-180:~/Downloads$ geturl 3500px-Mona_Lisa,_by_Leonardo_da_Vinci,_from_C2RMF_retouched.jpeg 
Enter your email to link or create your filepicker.io acccount: foo@bar.com

Uploading the file to Filepicker.io...

curl: (26) couldn't open file "3500px-Mona_Lisa"

Traceback (most recent call last):
  File "/usr/local/bin/geturl", line 64, in <module>
    {"filename": filename, "apikey": APIKEY, "fpurl": FPURL, "fileurl": fileurl}, shell=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 537, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command 'curl --progress-bar -F "fileUpload=@3500px-Mona_Lisa,_by_Leonardo_da_Vinci,_from_C2RMF_retouched.jpeg" -F "apikey=AFC21lQ73SYmfQT2Ya4oSz" https://www.filepicker.io/api/path/storage/3500px-Mona_Lisa%2C_by_Leonardo_da_Vinci%2C_from_C2RMF_retouched.jpeg' returned non-zero exit status 26
```

**Comment by peterbe** [comment]
Some escaping on the filename would help

**Comment by uams** [comment]
This should be fixed now thanks to #15  Thanks.
